
package callapp;


message Class1{
    enum TYPE{
        kUnknown = 0;
        kType1   = 1;
        kType2   = 2;
    };
    optional string name = 1;
    optional int32  age  = 2;
    optional int64  ts   = 3;
    repeated string name_list = 4;
    repeated bool   succ_list = 5;
    repeated TYPE   type_list = 6;
};


message Class2{
    optional string name = 1;
    optional uint32 age  = 2;
    optional uint64 ts   = 3;
    repeated uint32 age_list = 4;
};


message AllClass{
    optional Class1 cls1 = 1;
    repeated Class2 cls2 = 2;
};


